quizapp/src/main/java/billennium/quizapp/controller/CandidateController.java

```diff
@@ -21,4 +27,11 @@
     public String getUserId(@PathVariable("email") String email) {
         return candidateService.findByEmail(email);
     }
+
+    @PostMapping(path = SAVE_CANDIDATE,
+            consumes = APPLICATION_JSON_VALUE)
+    @ResponseStatus(HttpStatus.CREATED)
+    public void saveUser(@RequestBody String email) {
```

Comment:
warto aby POST'y od zapisu zwracały to co zostało zapisane. Często gęsto wykorzystuje się to na frontendzie. Przykład: zapisuje coś w bazie a potem chce zrobić od razu redirect do tego obiektu na froncie - a do tego potrzebuje ID bo routing mój działa po właśnie ID obiektu. @SzymonKajdan ważna informacja też dla Ciebie